app/perftools/management/commands/create_page_cache.py

```diff
@@ -329,6 +329,7 @@ def create_results_cache():
             print(f"- executing {keyword}")
             data = build_stat_results(keyword)
             print("- creating")
+            del data['hackathons']
```

Comment:
Just to be on the safe side:

```suggestion
            if 'hackathons' in data:
                    del data['hackathons']
```